cores/arduino/SERCOM.cpp

```diff
@@ -413,6 +413,12 @@ void SERCOM::disableWIRE()
   }
 }
 
+void SERCOM::runstandbyWIRE(bool runstdby) {
+  disableWIRE();                                 //  RUNSTDBY bit is enable protected so disable the peripheral
+  sercom->I2CM.CTRLA.bit.RUNSTDBY = runstdby ;
```

Comment:
Is there any harm in setting this bit all the time on enable?

I'd prefer not to add platform specifics API's to the Wire lib.

Reply:
There might be an increase in power consumption when RUNSTDBY is enabled.